test/common/router/router_test.cc

```diff
@@ -4725,6 +4725,7 @@ TEST_F(RouterTest, PropagatesUpstreamFilterState) {
   EXPECT_TRUE(verifyHostUpstreamStats(1, 0));
 
   EXPECT_TRUE(filter_state_verified);
+  EXPECT_TRUE(callbacks_.streamInfo().upstreamFilterState()->hasDataWithName("upstream data"));
```

Comment:
Can you update the comment on this test to make it clear that we're also verifying that it gets set on the connection FS?

Reply:
Done.